internal/services/securitycenter/security_center_subscription_pricing_resource.go

```diff
@@ -176,3 +212,26 @@ func resourceSecurityCenterSubscriptionPricingDelete(d *pluginsdk.ResourceData,
 	log.Printf("[DEBUG] Security Center Subscription deletion invocation")
 	return nil
 }
+
+func ConvertInputExtensionToSDKModel(inputList []interface{}) *[]pricings_v2023_01_01.Extension {
```

Comment:
```suggestion
func expandSecurityCenterSubscriptionPricingExtensions(inputList []interface{}) *[]pricings_v2023_01_01.Extension {
```

we tend to call these functions "expand..." so it would be good to update the name here just for consistency.

Reply:
Done